shared/js/ui/templates/shared/render-broken-site-href.es6.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (browser, url) {
```

Comment:
This is used by both the main popup and the hamburger menu, so I thought it makes more sense as a shared template